vue/src/components/HelloWorld.vue

```diff
@@ -2,14 +2,12 @@
   <div class="hello">
     <h1>{{ msg }}</h1>
     <p>
-      For a guide and recipes on how to configure / customize this project,<br>
-      check out the
+      For a guide and recipes on how to configure / customize this project, check out the
```

Comment:
Forced line break looks strange in narrow windows (such as in the editor's preview pane).